nvchecker_source/regex.py

```diff
@@ -22,9 +22,7 @@ async def get_version_impl(info):
 
   res = await session.get(conf['url'])
   body = res.body.decode(encoding)
-  try:
-    version = regex.findall(body)
-  except ValueError:
-    if not conf.get('missing_ok', False):
-      raise GetVersionError('version string not found.')
+  version = regex.findall(body)
+  if not len(version) and not conf.get('missing_ok', False):
```

Comment:
```suggestion
  if not version and not conf.get('missing_ok', False):
```

I also suggest to rename `version` to `versions`.

Reply:
Thanks! Done.